locales/en.default.schema.json

```
@@ -1273,6 +1273,9 @@
         "enable_sticky_info": {
           "label": "Enable sticky product information on large screens"
         },
+        "hide_variants": {
+          "label": "Hide other variants’ media when a variant is selected"
```

If the expectation is to have all variant media visible inside the modal, does this setting naming still makes sense? My expectation is that turning this on will hide other variants' media EVERYWHERE.

---

```suggestion
          "label": "Hide other variants’ media after selecting a variant"
```

Hey @gretchen-willenberg the reason why we didn't see yet this line of text is because the PR was not merged yet. I suggested the change based on your audit!